srcpkgs/iamb/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'iamb'
+pkgname=iamb
+version=0.0.9
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+makedepends="oniguruma-devel sqlite-devel"
+short_desc="Matrix client for Vim addicts"
+maintainer="Bnyro <bnyro@tutanota.com>"
+license="Apache-2.0"
+homepage="https://iamb.chat/"
+distfiles="https://github.com/ulyssa/iamb/archive/refs/tags/v${version}.tar.gz"
+checksum=7ef6d23a957bfab62decd48caa83c106a49d95760b4b2ccf5a6b6a8f4506e687
```

Comment:
The .desktop file, example config, and man pages could be included here as well
https://github.com/ulyssa/iamb/blob/main/PACKAGING.md#documentation
(not sure about the icons?)
```suggestion
checksum=7ef6d23a957bfab62decd48caa83c106a49d95760b4b2ccf5a6b6a8f4506e687

post_install() {
	vinstall iamb.desktop 644 usr/share/applications
	vman docs/iamb.1
	vman docs/iamb.5
	vsconf config.example.toml
}
```

Reply:
Oh yes, I completely forgot about looking up if there are any static files we could install.

I think we should install the icons too, I've seen some other packages doing the same thing.